ecommerce/auth/json_token.py

```diff
@@ -2,11 +2,12 @@
 import os
 from jose import jwt
 
+
 SECRET_KEY = os.environ["SECRET_KEY"]
 REFRESH_SECRET_KEY = os.environ["REFRESH_SECRET_KEY"]
 ALGORITHM = "HS256"
 ACCESS_TOKEN_EXPIRE_MINUTES = 30
-REFRESH_TOKEN_EXPIRE_MINUTES = 60 * 24
+REFRESH_TOKEN_EXPIRE_MINUTES = 60 * 24 * 30
 
 
 def create_access_token(subject: str):

```

Comment:
```suggestion
def create_access_token(subject: str) -> str:
```

Argument is typed, but return value not, keep consistent typing